redfish-core/lib/oem/ibm/lamp_test.hpp

```diff
@@ -131,14 +131,26 @@ inline void setLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
             BMCWEB_LOG_DEBUG << "GetObjectType: " << service;
 
             crow::connections::systemBus->async_method_call(
-                [aResp](const boost::system::error_code ec) {
+                [aResp, state](const boost::system::error_code ec) {
                     if (ec)
                     {
                         BMCWEB_LOG_DEBUG
                             << "Can't set lamp test status. Error: " << ec;
                         messages::internalError(aResp->res);
                         return;
                     }
+                    crow::connections::systemBus->async_method_call(
+                        [aResp](const boost::system::error_code ec) {
+                            if (ec)
+                            {
+                                BMCWEB_LOG_DEBUG << "Panel Lamp test failed."
+                                                 << ec;
+                                messages::internalError(aResp->res);
+                                return;
+                            }
+                        },
+                        "com.ibm.PanelApp", "/com/ibm/panel_app",
+                        "com.ibm.panel", "TriggerPanelLampTest", bool(state));
                 },
                 service, "/xyz/openbmc_project/led/groups/lamp_test",

```

Comment:
Why can't this application listen for this ?

Reply:
Lamp Test api present in led manager does not do the lamp testing for panel lcd blocks (where there are back leds present for each of the panel lcd block). For that to happen a lamp test command needs to be sent to the panel's micro code. "TriggerPanelLampTest" dbus method does the latter.

so whenever there is a lamp test request initiated (from bmcweb/from panel) both - led manager's lamp test and panel lamp test command needs to be executed. thats why we have added this portion of panel lamp testing.

Reply:
@gtmills : Let me elaborate a little - We came to this design after discussions with Vishwa and @lkammath , here's the rationale:

There is a use-case where one needs to restart lamp test. The LED manager handles a repeated "set" to the asserted property to restart the lamp test timer it runs. In such as case, property changed signal does not work because the property goes from true to true.

There were two options:
* Have the LED manager make a call to the panel app (this would be downstream code)
* Add another D-Bus call to the panel app from code in bmcweb that is anyway downstream only (this PR).

We chose the latter.

Reply:
I don't like this. I added this to our list to revisit for P11. 
I feel the Lamp Test Interface doesn't make a lot of sense as a bool. A true to true reset the timer. 
That said, approved, as we discussed, a lot of code wrote against it already.